src/components/WalletCongresSenate/Voters/index.js

```diff
@@ -2,26 +2,42 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useSelector } from 'react-redux';
 import List from 'antd/es/list';
+import Avatar from 'antd/es/avatar';
+import Flex from 'antd/es/flex';
 import { identitySelectors } from '../../../redux/selectors';
 import CopyIconWithAddress from '../../CopyIconWithAddress';
+import { getAvatarParameters } from '../../../utils/avatar';
 
 function Voters({ voting }) {
   const names = useSelector(identitySelectors.selectorIdentityMotions);
   return (
     <List
       dataSource={voting}
       locale={{ emptyText: 'No voting record found' }}
+      itemLayout="vertical"
       renderItem={(item) => {
         const id = item.toString();
         const identity = names?.[id]?.identity;
+        const { color, text } = getAvatarParameters(
+          identity?.name || identity?.legal || id,
+        );
         return (
-          <CopyIconWithAddress
-            isTruncate
-            name={identity?.name}
-            legal={identity?.legal}
-            address={id}
-            showAddress
-          />
+          <List.Item>
+            <Flex wrap align="center" gap="10px">
+              <Avatar size={50} style={{ backgroundColor: color }}>
+                {text}
+              </Avatar>
+              <Flex vertical gap="5px">
```

Comment:
we use this component in many places now. Should it be a component?

Reply:
Yes, done! :-)